docs/core/tools/dotnet-vstest.md

```diff
@@ -0,0 +1,125 @@
+---
+title: dotnet-vstest command | Microsoft Docs
+description: The dotnet-vstest command builds a project and all of its dependencies. 
+keywords: dotnet-vstest, CLI, CLI command, .NET Core
+author: guardrex
+ms.author: mairaw
+ms.date: 03/09/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 0e36c070-2242-41d3-96f1-aea0aca48d4d
+---
+
+# dotnet-vstest
+
+## Name
+
+`dotnet-vstest` - Runs tests from the specified files.
+
+## Synopsis
+
+`dotnet vstest <TEST_FILE_NAMES> [--Settings|/Settings:<Settings File>] [--Tests|/Tests:<Test Names>] [--TestAdapterPath|/TestAdapterPath] [--Platform|/Platform:<Platform type>] [--Framework|/Framework:<Framework Version>] [--Parallel|/Parallel] [--TestCaseFilter|/TestCaseFilter:<Expression>] [--logger|/logger:<Logger Uri/FriendlyName>] [-lt|--ListTests|/lt|/ListTests:<File Name>] [--ParentProcessId|/ParentProcessId:<ParentProcessId>] [--Port|/Port:<Port>] [--Diag|/Diag:<Path to log file>] [[--] <args>...]] [-?|--Help|/?|/Help]`
+
+## Description
+
+The `dotnet-vstest` command runs the `VSTest.Console` command-line application to run automated unit and coded UI application tests.
+
+## Arguments
+
+`TEST_FILE_NAMES`
+
+Run tests from the specified files. Separate multiple test file names by spaces.
+
+## Options
+
+`--Settings|/Settings:<Settings File>`
+
+Settings to use when running tests.
+
+`--Tests|/Tests:<Test Names>`
+
+Run tests with names that match the provided values. Separate multiple values with commas.
+
+`--TestAdapterPath|/TestAdapterPath`
+
+Use custom test adapters from a given path (if any) in the test run.
+
+`--Platform|/Platform:<Platform type>`
+
+Target platform architecture used for test execution. Valid values are `x86`, `x64`, and `ARM`.
+
+`--Framework|/Framework:<Framework Version>`
+
+Target .NET Framework version used for test execution. Examples of valid values are `.NETFramework,Version=v4.6`, `.NETCoreApp,Version=v1.0`, etc. Other supported values are `Framework35`, `Framework40`, `Framework45`, and `FrameworkCore10`.
+
+`--Parallel|/Parallel`
+
+Execute tests in parallel. By default, all available cores on the machine are available for use. Set an explicit number of cores with a settings file.
+
+`--TestCaseFilter|/TestCaseFilter:<Expression>`
+
+Run tests that match the given expression. `<Expression>` is of the format `<property>Operator<value>[|&<Expression>]`, where Operator is one of `=`, `!=`, or `~`.  Operator `~` has 'contains' semantics and is applicable for string properties like `DisplayName`. Parenthesis `()` are used to group sub-expressions.
+
+`-?|--Help|/?|/Help`
+
+Prints out a short help for the command.
+
+`--logger|/logger:<Logger Uri/FriendlyName>`
+
+Specify a logger for test results.  
+      
+To publish test results to Team Foundation Server, use the `TfsPublisher` logger provider:
```

Comment:
wondering if these sentences should be indented/formatted differently so that the options are not mixed with the examples.